src/components/Content/Pursuance/views/MapView.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import {ForceGraph, ForceGraphNode, ForceGraphLink} from 'react-vis-force';
+
+export default class MapView extends Component {
+  render() {
+    return (
+      <ForceGraph simulationOptions={{ height: 300, width: 300 }}>
+        <ForceGraphNode node={{ id: 'first-node' }} fill="red" />
+        <ForceGraphNode node={{ id: 'second-node' }} fill="blue" />
+        <ForceGraphNode node={{ id: 'third-node' }} fill="green" />
+        <ForceGraphLink link={{ source: 'first-node', target: 'second-node' }} />
+        <ForceGraphLink link={{ source: 'third-node', target: 'second-node' }} />
+      </ForceGraph>
```

Comment:
@colinfruit Great! Can you grab all the pursuances (I think they should all be in redux already) and show connections between some of them?  How about making the side of each node equal to the length of the name of the pursuance, just so we can get a sense of the different options we have re: visuals?